automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -21,13 +21,13 @@
 
 import argparse
 import asyncio
+import logging
 import sys
 import traceback
 from datetime import datetime
 from pathlib import Path
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, Set
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)